code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -276,3 +277,12 @@
 		playsound(user.loc, "punchmiss", 50, 1)
 		return PROJECTILE_STOPPED
 	return FALSE
+
+
+/*
+/obj/item/gun/proc/handle_click_empty(mob/user)
+	if(user)
+		to_chat(user, SPAN_DANGER("*click*"))
+	else
+		src.visible_message("*click click*")
+	playsound(loc, 'sound/weapons/empty.ogg', 100, 1) */
```

Comment:
why is this commented out? If it's not functional, delete it. Also - pointless whitespace above.

Reply:
Leftover from 2 weeks ago, just forgor